lib/recaptcha/client_helper.rb

```diff
@@ -1,35 +1,43 @@
 module Recaptcha
   module ClientHelper
-    # Your public API can be specified in the +options+ hash or preferably
-    # using the Configuration.
-    def recaptcha_tags(options = {})
-      if options.key?(:stoken)
-        raise(RecaptchaError, "Secure Token is deprecated. Please remove 'stoken' from your calls to recaptcha_tags.")
-      end
-      if options.key?(:ssl)
-        raise(RecaptchaError, "SSL is now always true. Please remove 'ssl' from your calls to recaptcha_tags.")
-      end
-
+    def build_html(options = {})
```

Comment:
this is included everywhere, so needs a less generic name ... preferably starting with recaptcha_